// @calcom/atoms@1.0.47/dist/index-Tgv65804.js
import { memo as j, useState as T } from "react";
import { am as k, aG as g, ap as t, aH as U, aI as N, aJ as C, at as b, aq as n, aC as v } from "./index-18iX6pCF.js";
import { C as A } from "./Card-yc1yPtA2.js";
const I = [{
  id: 13,
  thumbnailUrl: "https://img.youtube.com/vi/ihE8OYTdklg/0.jpg",
  mediaLink: "https://go.cal.com/ooo",
  title: "ooo.new",
  description: "Easily go out-of-office",
  href: "https://ooo.new"
}, {
  id: 12,
  thumbnailUrl: "https://cal.com/og-image-cal-ai.jpg",
  mediaLink: "https://go.cal.com/cal-ai",
  title: "Cal.ai",
  description: "Your personal AI scheduling assistant",
  href: "https://go.cal.com/cal-ai"
}, {
  id: 11,
  thumbnailUrl: "https://img.youtube.com/vi/KTg_qzA9NEc/0.jpg",
  mediaLink: "https://go.cal.com/insights",
  title: "Insights",
  description: "Get a better understanding of your business",
  href: "https://go.cal.com/insights"
}, {
  id: 10,
  thumbnailUrl: "https://img.youtube.com/vi/jvaBafzVUQc/0.jpg",
  mediaLink: "https://go.cal.com/video",
  title: "Cal Video",
  description: "Free video conferencing with recording",
  href: "https://go.cal.com/video"
}, {
  id: 9,
  thumbnailUrl: "https://img.youtube.com/vi/93iOmzHieCU/0.jpg",
  mediaLink: "https://go.cal.com/round-robin",
  title: "Round-Robin",
  description: "Create advanced group meetings with round-robin",
  href: "https://go.cal.com/round-robin"
}, {
  id: 8,
  thumbnailUrl: "https://img.youtube.com/vi/piKlAiibAFo/0.jpg",
  mediaLink: "https://go.cal.com/workflows",
  title: "Automate Workflows",
  description: "Make time work for you and automate tasks",
  href: "https://go.cal.com/workflows"
}, {
  id: 7,
  thumbnailUrl: "https://img.youtube.com/vi/UVXgo12cY4g/0.jpg",
  mediaLink: "https://go.cal.com/routing-forms",
  title: "Routing Forms",
  description: "Ask questions and route to the correct person",
  href: "https://go.cal.com/routing-forms"
}, {
  id: 6,
  thumbnailUrl: "https://img.youtube.com/vi/yGiZo1Ry5-8/0.jpg",
  mediaLink: "https://go.cal.com/recurring-video",
  title: "Recurring Bookings",
  description: "Learn how to create a recurring schedule",
  href: "https://go.cal.com/recurring-video"
}, {
  id: 5,
  thumbnailUrl: "https://img.youtube.com/vi/0v_nQtpxC_4/0.jpg",
  mediaLink: "https://go.cal.com/payments-video",
  title: "Accept Payments",
  description: "Charge for your time with Cal.com's Stripe App",
  href: "https://app.cal.com/apps/stripe"
}, {
  id: 4,
  thumbnailUrl: "https://img.youtube.com/vi/zGr_s-fG84k/0.jpg",
  mediaLink: "https://go.cal.com/confirmation-video",
  title: "Requires Confirmation",
  description: "Learn how to be in charge of your bookings",
  href: "https://cal.com/resources/feature/opt-in"
}, {
  id: 3,
  thumbnailUrl: "https://img.youtube.com/vi/c7ZKFuLy1fg/0.jpg",
  mediaLink: "https://go.cal.com/routing-video",
  title: "Routing Forms, Workflows",
  description: "Ask screening questions of potential bookers to connect them with the right person",
  href: "https://cal.com/blog/cal-v-1-8"
}, {
  id: 2,
  thumbnailUrl: "https://img.youtube.com/vi/EAc46SPL6iA/0.jpg",
  mediaLink: "https://go.cal.com/teams-video",
  title: "How to set up Teams",
  description: "Learn how to use round-robin and collective events.",
  href: "https://cal.com/docs/enterprise-features/teams"
}, {
  id: 1,
  thumbnailUrl: "https://img.youtube.com/vi/60HJt8DOVNo/0.jpg",
  mediaLink: "https://go.cal.com/dynamic-video",
  title: "Dynamic booking links",
  description: "Booking link that allows people to quickly schedule meetings.",
  href: "https://cal.com/blog/cal-v-1-9"
}], d = I.slice(0).reverse();
function S() {
  const {
    t: r
  } = k(), [c, p] = T(() => {
    if (typeof window > "u")
      return d;
    try {
      const e = g.getItem("removedTipsIds");
      if (e !== null) {
        const i = e.split(",").map((o) => parseInt(o, 10));
        return d.filter((o) => i.indexOf(o.id) === -1);
      } else
        return d;
    } catch {
      return d;
    }
  }), h = (e) => {
    p((i) => {
      const s = g.getItem("removedTipsIds") || "", o = i.findIndex((l) => l.id === e);
      return g.setItem("removedTipsIds", `${i[o].id.toString()}${s.length > 0 ? `,${s.split(",")}` : ""}`), i.splice(o, 1), [...i];
    });
  }, a = c.slice(0).reverse();
  return /* @__PURE__ */ t("div", {
    className: "hidden pb-4 pt-8 lg:grid",
    style: {
      gridTemplateColumns: "1fr"
    },
    children: c.map((e) => /* @__PURE__ */ t("div", {
      className: "relative",
      style: {
        gridRowStart: 1,
        gridColumnStart: 1
      },
      children: /* @__PURE__ */ t("div", {
        className: "relative",
        style: {
          transform: `scale(${1 - a.indexOf(e) / 20})`,
          top: -a.indexOf(e) * 10,
          opacity: `${1 - a.indexOf(e) / 7}`
        },
        children: /* @__PURE__ */ t(A, {
          variant: "SidebarCard",
          thumbnailUrl: e.thumbnailUrl,
          mediaLink: e.mediaLink,
          title: e.title,
          description: e.description,
          learnMore: {
            href: e.href,
            text: r("learn_more")
          },
          actionButton: {
            onClick: () => h(e.id),
            child: r("dismiss")
          }
        })
      })
    }, e.id))
  });
}
const q = j(S);
function F({
  title: r,
  description: c,
  background: p,
  features: h,
  buttons: a,
  isParentLoading: e,
  children: i,
  plan: s
}) {
  const {
    activeTheme: o
  } = U(), {
    t: l
  } = k(), {
    isPending: y,
    hasTeamPlan: w
  } = N(), {
    data: u
  } = C.viewer.teams.getUpgradeable.useQuery(), f = `${p}${o === "dark" ? "-dark" : ""}.jpg`, x = !1, L = !!(u != null && u[0]);
  return s === "team" && (w || L) || s === "enterprise" && x ? i : y ? /* @__PURE__ */ t(b, {
    children: e
  }) : /* @__PURE__ */ n(b, {
    children: [/* @__PURE__ */ n("div", {
      className: "relative flex min-h-[295px] w-full items-center justify-between overflow-hidden rounded-lg pb-10",
      children: [/* @__PURE__ */ n("picture", {
        className: "absolute min-h-[295px] w-full rounded-lg object-cover",
        children: [/* @__PURE__ */ t("source", {
          srcSet: f,
          media: "(prefers-color-scheme: dark)"
        }), /* @__PURE__ */ t("img", {
          className: "absolute min-h-[295px] w-full select-none rounded-lg object-cover object-left md:object-center",
          src: f,
          loading: "lazy",
          alt: r
        })]
      }), /* @__PURE__ */ n("div", {
        className: "relative my-4 px-8 sm:px-14",
        children: [/* @__PURE__ */ t("h1", {
          className: v("font-cal text-3xl"),
          children: l(r)
        }), /* @__PURE__ */ t("p", {
          className: v("mb-8 mt-4 max-w-sm"),
          children: l(c)
        }), a]
      })]
    }), /* @__PURE__ */ t("div", {
      className: "mt-4 grid-cols-3 md:grid md:gap-4",
      children: h.map((m) => /* @__PURE__ */ n("div", {
        className: "bg-muted mb-4 min-h-[180px] w-full rounded-md  p-8 md:mb-0",
        children: [m.icon, /* @__PURE__ */ t("h2", {
          className: "font-cal text-emphasis mt-4 text-lg",
          children: m.title
        }), /* @__PURE__ */ t("p", {
          className: "text-default",
          children: m.description
        })]
      }, m.title))
    })]
  });
}
export {
  q as Tips,
  F as UpgradeTip
};
