// @calcom/atoms/dist/DatePicker-m86EibR7.js
import { am as f, av as g, aw as y, ax as u, ap as x, ay as A, az as M, aA as j } from "./index-18iX6pCF.js";
const b = ({
  event: n,
  schedule: e,
  classNames: t,
  scrollToTimeSlots: C
}) => {
  var D, k, d, P, Y;
  const {
    i18n: S
  } = f(), [r, a] = g((o) => [o.month, o.selectedDate], y), [i, p, w] = g((o) => [o.setSelectedDate, o.setMonth, o.setDayCount], y), T = u((D = e == null ? void 0 : e.data) == null ? void 0 : D.slots);
  return /* @__PURE__ */ x(A, {
    customClassNames: {
      datePickerTitle: t == null ? void 0 : t.datePickerTitle,
      datePickerDays: t == null ? void 0 : t.datePickerDays,
      datePickersDates: t == null ? void 0 : t.datePickerDate,
      datePickerDatesActive: t == null ? void 0 : t.datePickerDatesActive,
      datePickerToggle: t == null ? void 0 : t.datePickerToggle
    },
    className: t == null ? void 0 : t.datePickerContainer,
    isPending: e.isPending,
    onChange: (o) => {
      i(o === null ? o : o.format("YYYY-MM-DD"));
    },
    onMonthChange: (o) => {
      p(o.format("YYYY-MM")), i(o.format("YYYY-MM-DD")), w(null);
    },
    includedDates: T,
    locale: S.language,
    browsingDate: r ? M(r) : void 0,
    selected: M(a),
    weekStart: j((P = (d = (k = n == null ? void 0 : n.data) == null ? void 0 : k.users) == null ? void 0 : d[0]) == null ? void 0 : P.weekStart),
    slots: (Y = e == null ? void 0 : e.data) == null ? void 0 : Y.slots,
    scrollToTimeSlots: C
  });
};
export {
  b as DatePicker
};
