// .ignored/@calcom/atoms/dist/PoweredBy-FQ2NbUs-.js
import { am as r, an as m, ao as d, ap as a, aq as e, ar as x, as as h, at as t, au as i } from "./index-18iX6pCF.js";
const u = ({
  logoOnly: l
}) => {
  const {
    t: n
  } = r(), s = m.useSession(), o = d(), c = s.data ? s.data.hasValidLicense : null;
  return /* @__PURE__ */ a("div", {
    className: `p-2 text-center text-xs sm:text-right${o ? " max-w-3xl" : ""}`,
    children: /* @__PURE__ */ e(x, {
      href: h,
      target: "_blank",
      className: "text-subtle",
      children: [!l && /* @__PURE__ */ e(t, {
        children: [n("powered_by"), " "]
      }), i === "Cal.com" || !c ? /* @__PURE__ */ a(t, {
        children: /* @__PURE__ */ a("img", {
          className: "relative -mt-px inline h-[10px] w-auto dark:invert",
          src: "/api/logo",
          alt: "Cal.com Logo"
        })
      }) : /* @__PURE__ */ a("span", {
        className: "text-emphasis font-semibold opacity-50 hover:opacity-100",
        children: i
      })]
    })
  });
};
export {
  u as default
};
